src/sidebar/services/load-annotations.ts

```diff
@@ -175,6 +183,41 @@ export class LoadAnnotationsService {
     this._searchClient.get({ group: groupId, uri: uris });
   }
 
+  /**
+   * If anchoring for an annotation the current set of annotations in the store
+   * fails to complete in a reasonable amount of time, then we assume that the
+   * annotation failed to anchor. If it does later successfully anchor then the
+   * status will be updated.
+   */
+  private _scheduleAnchorStatusCheck() {
+    const ANCHORING_TIMEOUT = 500;
+
+    const anchoringIds = this._store
+      .allAnnotations()
+      .filter(isWaitingToAnchor)
```

Comment:
I would recommend passing the `ann` argument explicitly here to avoid a hazard where an extra optional argument is added to `isWaitingToAnchor` in future and the second argument from `filter` gets passed by mistake.